Tests/FunctionalTests/Runner/TestValidation.js

```diff
@@ -36,11 +37,16 @@ module.exports.TestValidation = class TestValidation {
                         return false;
                     }
 
-                    correlationId = requestTelemetry.tags["ai.operation.id"];
+                    correlationId = requestTelemetry.data.baseData.id;
```

Comment:
Functional tests are using W3C propagation now, so ids & parent ids will always have the format `|<traceId>.<spanId>.`. Previously, this stuff depended on the child's parent ids being `|<traceId>.` which can no longer happen. A spanId is always included!